app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,67 @@
+class TweetController < ApplicationController
+	
+	get '/tweets' do 
+		@tweets = Tweet.all 
+		auth_erb('tweets/tweets')
+	end
+	
+	get '/tweets/new' do 
+		auth_erb('tweets/create_tweet')
+	end
+	
+	post '/tweets/new' do 
+		tweet = Tweet.new(:content => params[:content])
+		if valid_tweet?(tweet)
+			tweet.save
+			current_user.tweets << tweet
+			current_user.save
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.